resources/Kconif.diff

```diff
@@ -0,0 +1,5 @@
+84a85,88
+> config LOGO_NODEOS_CLUT224
+> 	bool "Gentoo-ised 224-colour logo"
```

Comment:
Nodeos-ized would makes sense if it's a modified version of Tux, but being a totally new logo, NodeOS standalone is just enough... :-P But nodeos-ized is fine for now if you don't need to do more changes on the file :-)